src/front-end/src/components/ManageMentors.js

```diff
@@ -0,0 +1,117 @@
+            </font>
+          </h1>
+          <br />
+          <br />
+          <div id="CurrentActiveMentors">
+            <h1>
+              <center>
+                <i>
+                  <font size="5"> Current/Active Mentors: </font>
+                </i>
+              </center>
+            </h1>
+            <p>
+              <font size="4">
+                <center>
+                  **(under development)**
+                  <br />
+                  Insert/pull current/active mentors here.
+                </center>
+              </font>
+            </p>
+          </div>
+          <div id="InvitedPendingMentors">
+            <h1>
+              <center>
+                <i>
+                  <font size="5"> Invited/Pending Mentors: </font>
+                </i>
+              </center>
+            </h1>
+            <p>
+              <font size="4">
+                <center>
+                  **(under development)**
+                  <br />
+                  Insert/pull invited/pending mentors here.
+                </center>
+              </font>
+            </p>
+          </div>
+          <div id="InviteTab">
+            <h1>
+              <center>
+                <i>
+                  <font size="5"> Invite A Mentor! </font>
+                </i>
+              </center>
+            </h1>
+            <p>
+              <font size="4">
+                <center>
+                  <i> How To Invite A Mentor: </i>
+                  <br />
+                  To invite a user to join Mentor Match as a mentor that is
+                  associated with your organization, type their email address in
+                  the text box below and then click the &apos;request&apos;
+                  button.
+                  <br />
+                  <br />
+                  The mentor will then receive a link via email which will take
+                  them to Mentor Match&apos;s Mentor Form.
+                  <br />
+                  <br />
+                  <Form.Item
+                    label="Mentor's Email Address:"
+                    id="inviteMentorEmail"
+                    // must have an input:
+                    rules={[
+                      {
+                        required: true,
+                        message: "Please enter the mentor's email",
+                      },
+                    ]}
+                  >
+                    <Input
+                      onInput={this.handleNameChange}
+                      id="inviteMentorEmail"
+                      placeholder="Email Address"
+                    />
+                  </Form.Item>
+                  <Button
+                    onClick={requestMentor}
+                    type="primary"
+                    htmlType="submit"
+                    className="formSubmitButton"
+                    size="large"
+                  >
+                    Request Mentor
+                  </Button>
```

Comment:
It might be useful to have the name field incase we need to store the requested Mentors in an array for admins in the database (maybe in a 'requestedMentors' field). I also think we should restrict the input to be an email address. In order to use the Form.Item tag, the code should be encapsulated within a Form tag (otherwise it won't check to validate if there is information in a required field) . The changes would be as follows:
```suggestion
                  <Form>
                    <Form.Item
                      label="Mentor's Email Address:"
                      id="inviteMentorEmail"
                      name="emailInput"
                      // must have an input:
                      rules={[
                        {
                          required: true,
                          message: "Please enter the mentor's email",
                        },
                      ]}
                    >
                      <Input
                        type="email"
                        onInput={this.handleNameChange}
                        id="inviteMentorEmail"
                        placeholder="Email Address"
                      />
                    </Form.Item>
                    <Button
                      onClick={requestMentor}
                      type="primary"
                      htmlType="submit"
                      className="formSubmitButton"
                      size="large"
                    >
                      Request Mentor
                    </Button>
                  </Form>
      
```

Reply:
Updated! Thanks for the feedback!